app/views/shared/provider/_page_title.html.slim

```diff
@@ -0,0 +1,2 @@
+- if content_for :title
```

Comment:
@thomasmaas @guicassolato Partial added ;)

Reply:
Though still repeating `content_for :title`.

Reply:
@damianpm @guicassolato @thomasmaas 
https://github.com/3scale/porta/pull/45#discussion_r220606974 🤔

Reply:
Well, that repetition is fixable by:
```
- if (title = content_for(:title))
  h1 = title
```
But my comment there is a whole other story 😄

Reply:
@damianpm 

```
- if (title = content_for(:title).presence)
  h1 = title
```

is a nice dry up of `content_for`. Lets do that.